[2025-01-31T01:41:07.937+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T01:41:07.971+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: terceira_dag.tsk1 manual__2025-01-31T01:41:05.585285+00:00 [queued]>
[2025-01-31T01:41:07.983+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: terceira_dag.tsk1 manual__2025-01-31T01:41:05.585285+00:00 [queued]>
[2025-01-31T01:41:07.985+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-31T01:41:08.002+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): tsk1> on 2025-01-31 01:41:05.585285+00:00
[2025-01-31T01:41:08.017+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3162) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T01:41:08.030+0000] {standard_task_runner.py:72} INFO - Started process 3164 to run task
[2025-01-31T01:41:08.030+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'terceira_dag', 'tsk1', 'manual__2025-01-31T01:41:05.585285+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/terceira_dag.py', '--cfg-path', '/tmp/tmp4g3_achu']
[2025-01-31T01:41:08.031+0000] {standard_task_runner.py:105} INFO - Job 35: Subtask tsk1
[2025-01-31T01:41:08.091+0000] {task_command.py:467} INFO - Running <TaskInstance: terceira_dag.tsk1 manual__2025-01-31T01:41:05.585285+00:00 [running]> on host 15928652fac1
[2025-01-31T01:41:08.200+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='terceira_dag' AIRFLOW_CTX_TASK_ID='tsk1' AIRFLOW_CTX_EXECUTION_DATE='2025-01-31T01:41:05.585285+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-31T01:41:05.585285+00:00'
[2025-01-31T01:41:08.201+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T01:41:08.219+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-31T01:41:08.220+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-01-31T01:41:08.233+0000] {subprocess.py:99} INFO - Output:
[2025-01-31T01:41:13.245+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-31T01:41:13.325+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T01:41:13.326+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=terceira_dag, task_id=tsk1, run_id=manual__2025-01-31T01:41:05.585285+00:00, execution_date=20250131T014105, start_date=20250131T014107, end_date=20250131T014113
[2025-01-31T01:41:13.407+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T01:41:13.438+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-31T01:41:13.441+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
